documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -259,6 +259,82 @@
               description: 'Prometheus %(prometheusName)s has missed {{ printf "%%.0f" $value }} rule group evaluations in the last 5m.' % $._config,
             },
           },
+
+          {
+            alert: 'PrometheusScrapeSeriesCardinality',
+            expr: |||
+              sum(scrape_series_added) by (job) > 100 * 1000
```

Comment:
I think this would mean that 100,000 series would have to be added for each job at almost every interval to trigger? That seems unlikely to happen.